test/ejabberd_tests/tests/muc_light_SUITE.erl

```diff
@@ -232,19 +249,40 @@ disco_features(Config) ->
             escalus:assert(is_stanza_from, [?MUCHOST], Stanza)
         end).
 
+%% The room list is empty. Rooms_per_page set to `infinity`
+disco_rooms_empty_page_infinity(Config) ->
+    escalus:story(Config, [{alice, 1}], fun(Alice) ->
+        set_mod_config(rooms_per_page, infinity),
+        [] = get_disco_rooms(Alice)
+                                        end).
```

Comment:
Indentation. I know all auto indentation tools do it this way but in all our test cases (or at least most of them) we have the closing `end` for `escalus:story` fun on the same level as `escalus:story`. See the test case above.